content/millennium/r4/financial/charge-item.md

```diff
@@ -86,10 +86,15 @@ Search for ChargeItems that meet supplied query parameters:
  Name                  | Required?  | Type          | Description
 -----------------------|------------|---------------|-------------------------------------------------------
  `_id`                 | Yes        | [`token`]     | The logical resource id associated with the ChargeItem.
```

Comment:
Instead of Requrred? 'Yes' for _id and context, should we have 'This or context' for id and 'This or _id' for context?

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/665/commits/aa37a5294afd32516a75bd83c1abd309654fb0e9#diff-16d4f8f74626c62d57a8b71d6be699f60e72fffed8636bc32e6654d4252b3ebbR88-R89

Reply:
+1